cpp/ql/lib/semmle/code/cpp/models/interfaces/FormattingFunction.qll

```diff
@@ -121,16 +121,31 @@ abstract class FormattingFunction extends ArrayFunction, TaintFunction {
    * the first format specifier in the format string.
    */
   int getFirstFormatArgumentIndex() {
```

Comment:
I think the documentation needs to be updated to explain how implicit definitions are handled/ignored.